.kokoro/tests/run_tests_graalvm_native.sh

```diff
@@ -0,0 +1,47 @@
+#! /bin/bash
+# Copyright 2020 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# `-e` enables the script to automatically fail when a command fails
+set -e
+
+# Kokoro setup
+if [ -n "$KOKORO_GFILE_DIR" ]; then
+  # Move into project directory
+  cd github/cloud-sql-jdbc-socket-factory
+  # source secrets
+  source "${KOKORO_GFILE_DIR}/TEST_SECRETS.sh"
+  export GOOGLE_APPLICATION_CREDENTIALS="${KOKORO_GFILE_DIR}/testing-service-account.json"
+fi
+
+echo -e "******************** Installing modules... ********************\n"
+mvn install -DskipTests
+echo -e "******************** Installation complete.  ********************\n"
+echo -e "******************** Running tests... ********************\n"
+echo "JAVA_HOME: $JAVA_HOME"
+
+# Why "-Denforcer.skip"? It's because  enforcer complains about the specific
+# version of junit-platform-engine GraalVM requires.
+
+# Why change directories to run the tests? Because GraalVM test execution
+# requires at least one matching test per Maven module. Not all modules in this
+# repository have "*IntegrationTests.java".
+pushd jdbc/postgres
```

Comment:
There's no way to test the child modules from the parent module? 

Should we be testing all the modules then?

Reply:
I am trying few of them at first.